http-conduit/http-conduit.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:   >= 1.10
 name:            http-conduit
-version:         2.3.7.4
+version:         2.3.7.5
```

Comment:
Oh, now that I think of it, this should probably be 2.3.8 since this is adding a new API

Reply:
If you can make that change, I'd be happy to merge. Thanks!

Reply:
Cool, updated!